_posts/2023-04-13-denoising-diffusion/denoising-diffusion.Rmd

```diff
@@ -0,0 +1,208 @@
+---
+title: "Denoising Diffusion with torch"
+description: >
+  Currently, in generative deep learning, no other approach seems to outperform the family of diffusion models. Would you like to try for yourself? If so, our torch implementation of denoising diffusion  provides an easy-to-use, easy-to-configure interface.
+author:
+  - name: Daniel Falbel
+    affiliation: Posit
+    affiliation_url: https://www.posit.co/
+  - name: Sigrid Keydana
+    affiliation: Posit
+    affiliation_url: https://www.posit.co/
+slug: falbelkeydiffusion
+date: 2023-04-13
+bibliography: references.bib
+categories:
+  - Torch
+  - R
+  - Image Recognition & Image Processing
+  - Generative Models
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+preview: images/flowers.png
+editor_options: 
+  markdown: 
+    wrap: 72
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE, fig.width = 6, fig.height = 6)
+```
+
+#### *A Preamble, sort of*
+
+*As we're writing this -- it's April, 2023 -- it is hard to overstate
+the attention going to, the hopes associated with, and the fears
+surrounding deep-learning-powered image and text generation. Impacts on
+society, politics, and human well-being deserve more than a short,
+dutiful paragraph. We thus defer appropriate treatment of this topic to
+dedicated publications, and would just like to say one thing: The more
```

Comment:
Maybe a footnote with a couple of dedicated publications you might recommend? Doesn't have to/shouldn't be exhaustive, but it'd be nice to have.